tiled/adapters/files.py

```diff
@@ -106,6 +106,8 @@ class DirectoryAdapter(MapAdapter):
         # These are unofficial but common file extensions.
         ".hdf": "application/x-hdf5",
         ".hdf5": "application/x-hdf5",
+        # on opensuse csv -> text/x-comma-separated-values
```

Comment:
Is this necessary for prometheus? Should we satisfy the note about opensuse?

Reply:
Not necessary for prometheus. Just did this as a flyby.